### PR TITLE
new package: o

### PR DESCRIPTION
**Pull request by xyproto**
The related package request is here: https://github.com/termux/termux-packages/issues/4882

This is my first contribution to Termux so I hope I am following the correct procedure. Thanks for reviewing.

### CODE REVIEW

File: packages/o/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	go build
+}
+
+termux_step_make_install() {
+	install -Dm700 \
+		"$GOPATH"/src/github.com/xyproto/o/o \
+		"$TERMUX_PREFIX"/bin/
+	install -Dm600 \
+		"$TERMUX_PKG_SRCDIR"/o.1 \
+		"$TERMUX_PREFIX"/share/man/man1/
```
Comment (xeffyr) [inline packages/o/build.sh]: Use
```
install -Dm700 -t "$TERMUX_PREFIX"/bin "$GOPATH"/src/github.com/xyproto/o/o
install -Dm600 -t "$TERMUX_PREFIX"/share/man/man1 "$TERMUX_PKG_SRCDIR"/o.1
```

Comment (xeffyr) [reply packages/o/build.sh]: Fixed it myself.

**Comment by xyproto** [comment]
Thanks for merging!

**Comment by xeffyr** [comment]
Package is renamed to `o-editor` due to certain side effects of one-character name: https://github.com/aptly-dev/aptly/issues/908.

**Comment by xyproto** [comment]
An understandable move! I was not aware that such a short name could cause issues, but hopefully hardening the edge cases of the package manager will be a good thing in the long run.

**Pull request merged**